qiskit/providers/aer/noise/device/models.py

```diff
@@ -265,21 +265,35 @@ def _device_depolarizing_error(qubits,
     # with dim = 2 ** N for an N-qubit gate error.
 
     error = None
+    num_qubits = len(qubits)
+    dim = 2 ** num_qubits
+
+    # If the device reports an error_param greater than the maximum
+    # allowed for a depolarzing error model:
+    #   error_param <= dim / (dim + 1)
+    # we truncated the depolarizing error probability to the maximum value
+    if error_param is not None:
+        error_param = min(error_param, dim / (dim + 1))
```

Comment:
Should the truncating be done in `basic_device_gate_errors()` rather than `_device_depolarizing_error()`? Because the basic_device_gate_error explicitly assumes a depolarizing channel, however there could be more generic noise channels that result in higher gate errors (not due to depolarizing). It seems like the `_device_depolarizing_error()` is a more generic function, potentially useful in other non-basic noise models?

Reply:
Good idea